src/runtime/node/vm/script.ts

```diff
@@ -0,0 +1,23 @@
+import type vm from "node:vm";
+import { createNotImplementedError } from "src/runtime/_internal/utils";
+
+export class Script implements vm.Script {
```

Comment:
While it wouldn't work in worked, i was thinking maybe we could emulate it partially with `eval`? (or modern base64 with dynamic import also should work https://github.com/unjs/mlly#evalmodule only issue is async)

Reply:
Getting the compliance of `eval`'s implementation right will be tricky and in some cases impossible. Is there any prior art we could reuse? 

Maybe we could leave that out of this PR due to the complexity. I suspect that we'll need two implementations anyway - one that is CSP/workerd compliant and doesn't use eval, and one that uses eval.

Reply:
Yeah. Let me see how hard it would be totally agree to defer if not easy.

Re CSP compatibility, i think either way we throw an error right? (if unenv completely not implements `eval` shim or the runtime disallows eval)

Reply:
~> https://github.com/unjs/unenv/issues/236